CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.10.2)
+cmake_minimum_required(VERSION 3.27.2)
```

Comment:
For compatibility with Ubuntu 20..04, can this be **3.16.3**?

Otherwise, I can not accept this PR

Reply:
Will test and come back to you